app/services/claims/fetch_eligible_advocate_categories.rb

```diff
@@ -26,16 +26,8 @@ def agfs_reform_categories
       Settings.agfs_reform_advocate_categories
     end
 
-    # TODO: to be changed to solely use fee_scheme objects once rest of app cleared up
     def agfs_reform?
-      case claim.fee_scheme
-      when 'default'
-        false
-      when 'fee_reform'
-        true
-      else
-        claim.fee_scheme&.agfs? && claim.fee_scheme.version >= 10
-      end
+      claim.agfs_reform? || Offence.in_scheme_ten.include?(claim.offence)
```

Comment:
check/test whether the Offence check is needed here as claim.agfs_reform should be able to determine this itself via `FeeScheme.for_claim(self)` logic in `base_claim`

Reply:
Agree, @colinbruce might have more insight on this as this was something that was raised out of some API related submissions `:not_sure_if:` ?!